### PR TITLE
Mention My integration in Tesla Fleet

### PR DESCRIPTION
**Pull request by Bre77**
## Proposed change

Mention that the My Home Assistant integration is a pre requisite. 

Requested https://community.home-assistant.io/t/reinstate-tesla-integration/640577/42

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes # https://community.home-assistant.io/t/reinstate-tesla-integration/640577/42

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Clarified prerequisites for using the Tesla Fleet API integration, ensuring the "My Home Assistant" integration remains enabled for optimal functionality.

### CODE REVIEW

File: source/_integrations/tesla_fleet.markdown [hunk 1]
```diff
@@ -24,7 +24,7 @@ The Tesla Fleet API {% term integration %} exposes various sensors from Tesla ve
 
 ## Prerequisites
 
-You must have a [Tesla](https://tesla.com) account and a Tesla vehicle, PowerWall, Solar, or Wall Connector.
+You must have a [Tesla](https://tesla.com) account and a Tesla vehicle, PowerWall, Solar, or Wall Connector, and must not have disabled the [My Home Assistant](/integrations/my/) integration.
```
Comment (frenck) [inline source/_integrations/tesla_fleet.markdown:27]: How is that a prerequisite?

To my knowledge, we have absolutely nothing in Home Assistant that requires My Home Assistant. Therefore, I'm wondering how that happened in this case? (Or if it really is happening in this case).

../Frenck

Comment (Bre77) [reply source/_integrations/tesla_fleet.markdown:27]: I have to set the redirect URL in the application details registered with Tesla, so there is an external constraint making the "my" integration required for OAuth redirects to work.

Comment (frenck) [reply source/_integrations/tesla_fleet.markdown:27]: In that case, we need to provide details on how people can set those up themselves.

Comment (Bre77) [reply source/_integrations/tesla_fleet.markdown:27]: That's been merged into Dev and the docs for it are in next. This PR is just to reflect the current state and requirement for using the built in client ID.

Comment (Bre77) [reply source/_integrations/tesla_fleet.markdown:27]: https://github.com/home-assistant/core/pull/123324

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Bre77 :+1:

**Pull request merged**